joomla-projects/GSoC17_publishing_workflow
# Resolve issue 114

Pull Request for Issue 114 .

List view for states and transitions do not show the corresponding workflow name. So you do not know on which workflow you are working.
This PR adds the workflow name to the title. 

### Testing Instructions
Open the states list of a workflow, you cannot see on which workflow you are working
Apply the Patch - the workflow name is shown in the the title.

The same on transitions list view

![workflowname-after](https://user-images.githubusercontent.com/1035262/30251123-24361926-965a-11e7-8ccd-22d17e31c2a3.PNG)

### Documentation Changes Required
Yes, if there are screenshots
